server/models/userProfile.js

```diff
@@ -0,0 +1,61 @@
+const mongoose = require('mongoose')
+
+const userProfile = new mongoose.Schema(
+  {
+    firstName: {
+      type: 'string',
+      required: false
+    },
+    lastName: {
+      type: 'string',
+      required: false
+    },
+    gender: {
+      type: 'string',
+      required: false
+    },
+    birthDay: {
+      type: Date,
+      default: null
+    },
+    email: {
+      type: 'string',
+      required: false
+    },
+    phoneNumber: {
+      type: 'string',
+      required: false
+    },
+    address: {
+      type: 'string',
+      required: false
+    },
+    city: {
+      type: 'string',
+      required: false
+    },
+    country: {
+      type: 'string',
+      required: false
+    },
+    description: {
+      type: 'string',
+      required: false
+    },
+    available: { type: 'boolean', required: true },
+    availableDays: {
```

Comment:
Good job making the change here. Maybe call this field `availabityPeriod` and have another field `availabilityDays`

Eg: 
```
availabilityDays: {
type: 'string'
enum: ['Mon', 'Tue' and son on]
}
```

Reply:
@rajivtitus ,I have made the changes